CHANGELOG.md

```diff
@@ -1,16 +1,21 @@
 # Changelog
 
-## 0.10.0 - Unreleased
+## 0.10.0
 
 ### Vision
 - Added new `ConstructionCone` class to `DetectionClass`
 - Added method `set(visionManager:)` to `VisionPresentationViewController`
 - Improved delivery of camera parameters in `CameraVideoSource`
+- Fixed a crash on `MBVVisionReplayManagerNative` destroy
 - Fixed the bug with unrecorded session
+- Fix a bug when observers hash table was mutated while being enumerated
```

Comment:
Please use unified verb (fixed)

```suggestion
- Fixed a bug when hash table's observers were mutated while being enumerated
```
or
```suggestion
- Fixed a bug when hash table was mutated while being enumerated
```

Could you please review the description one more time?

Reply:
fixed